python/nav/web/utils.py

```diff
@@ -64,3 +71,59 @@ def wrapper(request, *args, **kwargs):
         return wrapper
 
     return wrap
+
+
+def generate_qr_code(url: str, name: str = "") -> io.BytesIO:
```

Comment:
I would suggest just replacing `name` with `caption` here, it sounds clearer to me.